diff -urN aa/muine-0.5.1/configure.in muine-0.5.1/configure.in
--- aa/muine-0.5.1/configure.in	2004-04-03 14:35:06.000000000 +0200
+++ muine-0.5.1/configure.in	2004-04-03 23:19:11.600370416 +0200
@@ -101,7 +101,7 @@
 AC_SUBST(MONO_FLAGS)
 
 dnl Intl
-ALL_LINGUAS="ca cs de es eu fr hr hu ko nl no pt pt_BR sr sr@Latn sv"
+ALL_LINGUAS="ca cs de es eu fr hr hu ko nb nl pt pt_BR sr sr@Latn sv"
 GETTEXT_PACKAGE=muine
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
