diff -aurN muine-0.5.1.orig/data/muine.desktop.in muine-0.5.1/data/muine.desktop.in
--- muine-0.5.1.orig/data/muine.desktop.in	2004-01-17 16:30:56.000000000 +0100
+++ muine-0.5.1/data/muine.desktop.in	2004-04-03 22:55:23.356875512 +0200
@@ -3,9 +3,11 @@
 Encoding=UTF-8
 _Name=Muine Music Player
 _Comment=Listen to your music
+Name[pl]=Odtwarzacz muzyczny Muine
+Comment[pl]=Słuchaj Twojej muzyki
 Exec=muine
 Icon=muine.png
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;Application;AudioVideo;X-Ximian-Main;X-Red-Hat-Base;
+Categories=AudioVideo;Player;
