diff -ruN muine-0.8.3./src/PlaylistWindow.cs muine-0.8.3/src/PlaylistWindow.cs
--- muine-0.8.3./src/PlaylistWindow.cs	2005-05-01 18:25:15.000000000 +0200
+++ muine-0.8.3/src/PlaylistWindow.cs	2005-08-15 15:07:59.641730432 +0200
@@ -431,7 +431,7 @@
 		public void RestorePlaylist ()
 		{
 			// Load last playlist
-			if (!File.Exists (FileUtils.PlaylistFile))
+			if (!System.IO.File.Exists (FileUtils.PlaylistFile))
 				return;
 
 			OpenPlaylistInternal (FileUtils.PlaylistFile,
